src/frontends/onnx/tests/CMakeLists.txt

```diff
@@ -84,7 +85,12 @@ set(SRC
     model_support_tests.cpp
     onnx_ops_registration.cpp
     onnx_reader_external_data.cpp
-    skip_tests_config.cpp)
+    skip_tests_config.cpp
+    ../frontend/src/core/graph_iterator_proto.cpp
+    ../frontend/src/core/decoder_proto.cpp
+    ../frontend/src/core/graph_iterator_proto.hpp
+    ../frontend/src/core/decoder_proto.hpp
```

Comment:
I don't think you need to add hpp files in cmake

Reply:
Yeah, looks like it isn't necessary now, removed